### DISCUSSION TITLE
Node Paint brush tool.

### DISCUSSION THREAD

**Discussion by DarcTellier** [godotengine/godot-proposals#7864] [Editor]
A tool to copy /paste nodes  to a mouse position in a scene with left click, and deletes them with right click. 
Would make life easier to place coins and enemies.  With Same tools as TileMaps.

OR

Have nodes work in Tilemaps.

**Comment by dalexeev** [comment]
See:

* #5553
* [dalexeev/godot-node-brush-plugin](https://github.com/dalexeev/godot-node-brush-plugin) (3.x).

> Have nodes work in Tilemaps.

This is available in 4.x:

![](https://github.com/godotengine/godot-proposals/assets/47700418/e5e222e0-262d-40b8-818e-1e552ee88aa7)
